assemblies/creating-code-server-workbenches.adoc

```diff
@@ -0,0 +1,16 @@
+:_module-type: ASSEMBLY
+
+ifdef::context[:parent-context: {context}]
+
+[id="creating-code-server-workbenches_{context}"]
+= Creating code-server workbenches
+
+[role='_abstract']
+You can create a blank workbench or import a Jupyter notebook in code-server from several different sources.
```

Comment:
```suggestion
You can create a blank Jupyter notebook or import a Jupyter notebook in code-server from several different sources.
```

one more revision, I now see that this is just a code-server version of 

> You can create a blank notebook or import a notebook in JupyterLab from several different sources.

from assemblies/creating-and-importing-notebooks.adoc

The Jupyter notebook in question refers to "computational notebook document", https://docs.jupyter.org/en/latest/#what-is-a-notebook, not to a workbench.